plugins/InspectionGadgets/src/com/siyeh/ig/redundancy/RedundantStringOperationInspection.java

```diff
@@ -131,6 +135,54 @@ public void visitMethodCallExpression(@NotNull PsiMethodCallExpression call) {
       myProcessors.mapAll(call).forEach(myHolder::registerProblem);
     }
 
+    @Override
+    public void visitBinaryExpression(@NotNull PsiBinaryExpression expression) {
```

Comment:
Let's remove this part. This is already covered by "s.length() == 0 -> s.isEmpty()" conversion and "s.strip().isEmpty() -> s.isBlank()" conversion, so users can perform this in two steps. No need to maintain additional chunk of code to make this conversion in a single step.

Note that SizeReplaceableByIsEmpty will be on by default in the next IntelliJ IDEA version.

Reply:
Removed